CHANGELOG.md

```diff
@@ -39,7 +39,7 @@ Packages with dependency updates only:
 
 #### `flame` - `v1.4.0`
 
- - **REFACTOR**: move broadphase-related functionality into separate subdir ([#1943](https://github.com/flame-engine/flame/issues/1943)). ([f23acd41](https://github.com/flame-engine/flame/commit/f23acd41e0341909200437bfb6487cbe9ca58a53))
+ - **REFACTOR**: move broadphase-related functionality into separate subdirectory ([#1943](https://github.com/flame-engine/flame/issues/1943)). ([f23acd41](https://github.com/flame-engine/flame/commit/f23acd41e0341909200437bfb6487cbe9ca58a53))
```

Comment:
Why does it report on all these words that are already in the ignore list? :thinking:

Reply:
![image](https://user-images.githubusercontent.com/744771/195991531-34b76d0c-aa9f-418c-8f80-89d634da6262.png)

Reply:
this does not happens when i run cspell locally
may be there is some issue with the configuration file path